#164859765 user should bookmark articles

jmusila opened the pull request:
### What does this PR do?
- Adds articles as bookmarks to read later

#### Description of Task to be completed?
 - [x] write unit tests
 - [x] add an article to bookmarks
 - [x] remove an article from bookmarks 
- [x] get bookmarked articles

#### How should this be manually tested?
- Clone the repo
```$ git clone https://github.com/deferral/ah-django```

- Create a virtual environment with virtualenv
```$ virtualenv venv```

- Configure the database to work with the application
`$ python manage.py makemigrations`
`$ python manage.py migrate`

- Run the application
`$ python manage.py runserver`

- Testing Routes on Postman
- Adding to bookmark 
`/api/articles/<slug>/bookmark`
- Remove from bookmarks
` /api/articles/<slug>/bookmark`
- Get user bookmarked aticles
`api/bookmark/all`

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
#164859765

Screenshots (if appropriate)
![Insomnia – http:--127 0 0 1:8000-api-articles-how-to-be-start-writing_myweek-bookmark-_179](https://user-images.githubusercontent.com/42172425/55835509-fe9a8300-5b24-11e9-8293-e1602db94366.png)
![Insomnia – http:--127 0 0 1:8000-api-articles-how-to-be-start-writing_myweek-bookmark-_180](https://user-images.githubusercontent.com/42172425/55835523-05c19100-5b25-11e9-8a51-8324d48db509.png)

wangonya commented:
Tested. Works 👍🏾. You can also add getting the bookmarked articles by the user. It might not have been listed in the description but I think it will be required once we get to the frontend. I'm thinking users will want to see their bookmarked articles on their profiles. @philophilo thoughts?

jmusila committed:
164859765-feat(bookmark article): user should bookmark article

- a user should be able to bookmark article
- a user should be able to unbookmark article
- a user shoul be able to view bookmarked articles

[Delivers #164859765]: ft-bookmark-article-164859765
